test/nightly/azureResourceGraph.test.ts

```diff
@@ -23,22 +22,12 @@ suite('Graph action', async function (this: Mocha.Suite): Promise<void> {
             this.skip();
         }
         this.timeout(2 * 60 * 1000);
-        resourceGroupName = randomUtils.getRandomHexString(12);
-        // Cosmos DB account must have lower case name
-        accountName = randomUtils.getRandomHexString(12).toLowerCase();
+        resourceGroupName = resourceGrouList[api.Graph] === undefined ? '' : resourceGrouList[api.Graph];
```

Comment:
If we can't find the the graph account, we should throw an error rather than returning `''`.  That's just going to result in weird test errors.  But hopefully with the `require` method I was talking about, this won't be an issue, but we should still check regardless.